cmake/haiku.cmake

```diff
@@ -0,0 +1,4 @@
+# set compiler flags
+set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -lnetwork -lexecinfo")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O2 -g -std=c11 -D_BSD_SOURCE")
```

Comment:
`-O2 -q` should not be required, `-D_BSD_SOURCE` looks suspicious and the standard should not be set here. I'm also not aware of C11 features used.

Reply:
_BSD_SOURCE was required so far because Haiku had no features.h, but this patchset is almost 1 year old (i just had no time to upstream it), so thanks for the input, i will recheck.